src/apollo/queries.ts

```diff
@@ -265,6 +265,21 @@ export type EnsAccountRegistratonsData = {
   };
 };
 
+export const ENS_ALL_ACCOUNT_REGISTRATIONS = gql`
```

Comment:
added this back for registrations in ens intro screen, `ENS_ACCOUNT_REGISTRATIONS` is taking a limit 99 so it was failing to get all ens that an account has, for example rainbowwallet.eth